### PR TITLE
add pkg-config to test image

### PR DESCRIPTION
**Pull request by beniwohli**
newest release of mysqlclient requires pkg-config to build.

I also had to switch netcat to netcat-traditional, as the netcat package isn't available in the Python 3.11 base image

### CODE REVIEW

**Comment by beniwohli** [comment]
Bit of a catch-22 here. Can't merge this before tests pass, but tests won't pass before we merge this and docker images are rebuilt.

@basepi how about we disable the mysqlclient test in this PR, merge, wait for rebuild, and then re-enable the test in a follow-up PR?

**Comment by basepi** [comment]
> @basepi how about we disable the mysqlclient test in this PR, merge, wait for rebuild, and then re-enable the test in a follow-up PR?

Sounds workable to me.

**Review by basepi — APPROVED** [review]

**Pull request merged**
